app/assets/stylesheets/frontend/views/publications/_show.scss

```diff
@@ -0,0 +1,120 @@
+.publications-show-page {
+  p {
+    @include core-19;
+  }
+  .govspeak ul {
+    margin-left:0px;
+  }
+
+  .heading-block {
+    h1 {
+      @include media(tablet) {
+        padding-left: $gutter-half;
+        margin-bottom: $gutter-two-thirds;
+      }
+    }
+    .document-type {
+      @include core-19;
+      float: left;
+      margin-right: 5px;
+      @include media(tablet) {
+        @include bold-24;
+        float: none;
+      }
+    }
+    .document-size {
+      @include core-19($tabular-numbers: true);
+      float: left;
+      @include media(tablet) {
+        float: none;
+      }
+    }
+    .changed{
+      display: block;
+    }
+  }
+
+  .attachment {
+
+    padding: $gutter-one-third 0 0 0;
+
+    @include media(tablet) {
+      padding: $gutter-half 0 0 0;
+      @include right-to-left {
+        padding: $gutter-half 0 0 0;
+      }
+    }
+
+    .attachment-details {
+
+      @include grid-column( 2/3 );
+
+      h1, h2, h3 {
```

Comment:
as far as I can tell from the other files in this pull request only h2 is used within `attachment-details`. Are `h1` and `h3` needed?